steward/src/commands/fund_cellar.rs

```diff
@@ -0,0 +1,135 @@
+#![allow(unused_variables)]
```

Comment:
Your branch is re-adding these deleted commands, will have to remove this file and some others that were added back here.

Reply:
Bumping this, these files still need to be removed.